ams/ams-core/src/main/java/com/redhat/services/ffm/ams/core/models/TermRequest.java

```diff
@@ -0,0 +1,98 @@
+package com.redhat.services.ffm.ams.core.models;
+
+import java.util.Objects;
+
+public class TermRequest {
```

Comment:
TermsRequest, probably?

Reply:
Actually, I cannot see any usage of this class.

Reply:
Hi @carlesarnal , yes you are right. I've renamed to `TermsRequest`. 

This class is used in the `CreateResourceRequest`, in particular in the `createResource` method this is used to retrieve the terms and check if the user has accepted them